components/Homepage/Experience.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const Experience = () => {
+  return (
+    <section className='bg-white-900 dark:bg-gray-900'>
+      <h2>
+        Work
+        <div className='relative w-fit'>
+          <div className='relative z-10'>Experience</div>
+          <div className='absolute bottom-0.5 z-0 h-6 w-1/2 bg-[#FFBE62]' />
```

Comment:
It's better to not hardcode colors and add them to your tailwind config instead.